nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -143,10 +143,14 @@ $(CsWinRTFilters)
   <Target Name="CsWinRTPlaceWinMDInOutputFolder"
           Condition="'$(CsWinRTComponent)' == 'true'" 
           BeforeTargets="AfterBuild">
-    <Copy SourceFiles="Generated Files\$(AssemblyName).winmd"
-          DestinationFolder="$(TargetDir)"
-          UseHardlinksIfPossible="false"
-          SkipUnchangedFiles="true" />
+    <ItemGroup>
+      <CsWinRTGeneratedWinMD Include="$(MSBuildProjectDirectory)\Generated Files\net5**\$(AssemblyName).winmd"
```

Comment:
Consider CsWinRTGeneratedFilesDir

Reply:
Actually for now you might want GeneratedFilesDir as this is what source generator uses.